### PR TITLE
Feature/prep release 1.2.0

### PR DESCRIPTION
**Pull request by RainbowDashLabs**
Changes for the next release.

Changed the info command to include contributors and future artist responsible for branding and badges.

### CODE REVIEW

File: src/main/resources/version [hunk 1]
```diff
@@ -0,0 +1 @@
+${version}
```
Comment (dlsf) [inline src/main/resources/version:1]: Missing newline

**Review by dlsf — APPROVED** [review]

File: src/main/resources/version (hunk 1 above)
Comment (RainbowDashLabs) [reply src/main/resources/version:1]: I just read the file as a plain string. A new line would be removed by discord anyway 🤔

File: src/main/java/de/chojo/repbot/commands/Info.java [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+                        Replacement.create("TOS", configuration.links().tos()),
+                        Replacement.create("WEBSITE", configuration.links().website())) + "**", false)
+                .setColor(Colors.Pastel.BLUE)
+                .build();
+    }
+
+    @Override
+    public void onSlashCommand(SlashCommandEvent event) {
+        var eventWrapper = MessageEventWrapper.create(event);
+        event.reply(wrap(getResponse(eventWrapper))).queue();
```
Comment (DRSchlaubi) [inline src/main/java/de/chojo/repbot/commands/Info.java:113]: there is actually replyEmbeds or something like that now

**Review by DRSchlaubi — CHANGES_REQUESTED** [review]

File: src/main/java/de/chojo/repbot/commands/Info.java (hunk 2 above)
Comment (RainbowDashLabs) [reply src/main/java/de/chojo/repbot/commands/Info.java:113]: So much nicer :D

**Pull request merged**